app/views/spree/user_mailer/confirmation_instructions.text.erb

```diff
@@ -1,4 +0,0 @@
-<%= t('.welcome', email: @email) %>
-
-<%= t('.text') %>
-<%= @confirmation_url %>
```

Comment:
I don't really understand why this was added by 01746ed470 in the first place, but perhaps it would be good to keep the text version (but update the link). It's useful for accessibility right?